scipy/interpolate/__init__.py

```diff
@@ -5,87 +5,115 @@
 
 .. currentmodule:: scipy.interpolate
 
-Sub-package for objects used in interpolation.
+Sub-package for functions and objects used in interpolation.
+
+See the :ref:`user guide <tutorial-interpolate>` for recommendations on choosing a
+routine, and other usage details.
 
-As listed below, this sub-package contains spline functions and classes,
-1-D and multidimensional (univariate and multivariate)
-interpolation classes, Lagrange and Taylor polynomial interpolators, and
-wrappers for `FITPACK <http://www.netlib.org/dierckx/>`__
-and DFITPACK functions.
 
 Univariate interpolation
 ========================
 
 .. autosummary::
    :toctree: generated/
 
-   interp1d
+   make_interp_spline
+   CubicSpline
+   PchipInterpolator
+   Akima1DInterpolator
+   FloaterHormannInterpolator
    BarycentricInterpolator
    KroghInterpolator
-   barycentric_interpolate
-   krogh_interpolate
-   pchip_interpolate
    CubicHermiteSpline
-   PchipInterpolator
-   Akima1DInterpolator
-   CubicSpline
+
+**Low-level data structures for univariate interpolation:**
+
+.. autosummary::
+   :toctree: generated/
+
    PPoly
    BPoly
-   FloaterHormannInterpolator
+   BSpline
 
 
 Multivariate interpolation
 ==========================
 
-Unstructured data:
+**Unstructured data**
 
 .. autosummary::
    :toctree: generated/
 
-   griddata
    LinearNDInterpolator
    NearestNDInterpolator
    CloughTocher2DInterpolator
+   griddata - a convenience wrapper for the above three interpolators
    RBFInterpolator
-   Rbf
    interp2d
```

Comment:
The docs suggest that this is removed?
https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.interp2d.html#scipy.interpolate.interp2d

Reply:
Not quite. It follows a special-cased staged removal procedure: https://github.com/scipy/scipy/issues/18583
AFAIR it used to be listed in the deprecation tracker https://github.com/scipy/scipy/issues/15765, cannot find it in it anymore though.

Reply:
Ok, additional tools it is then, in the last commit.